petstore/clients/javascript/samples/typescript/README.md

```diff
@@ -0,0 +1,42 @@
+# client library samples for TypeScript 
+
+These sample programs show how to use the TypeScript client libraries for in some common scenarios.
+
+## Prerequisites
+
+The sample programs are compatible with [LTS versions of Node.js](https://github.com/nodejs/release#release-schedule).
+
+Before running the samples in Node, they must be compiled to JavaScript using the TypeScript compiler. For more information on TypeScript, see the [TypeScript documentation](https://www.typescriptlang.org/docs/home.html). Install the TypeScript compiler using:
+
+
+```bash
+npm install -g typescript
+```
+
+## Setup
+
+To run the samples using the client sdk:
```

Comment:
do we need to build generated code first?

Reply:
yes, I had updated the steps.